mbed/mbed-os.lib

```diff
@@ -0,0 +1 @@
+https://github.com/ARMmbed/mbed-os.git#26606218ad9d1ee1c8781aa73774fd7ea3a7658e

```

Comment:
What's special about that [commit](https://github.com/ARMmbed/mbed-os/commit/26606218ad9d1ee1c8781aa73774fd7ea3a7658e)? Can we not use master?

```suggestion
https://github.com/ARMmbed/mbed-os
```